build/Tests/UnitTests/CoreGraphics/CoreGraphics.UnitTests.vcxproj

```diff
@@ -239,6 +239,11 @@
     <ClangCompile Include="..\..\..\..\tests\unittests\CoreGraphics\CGContextTests.mm" />
     <ClangCompile Include="..\..\..\..\tests\unittests\CoreGraphics\CGBitmapContextTests.mm" />
     <ClangCompile Include="..\..\..\..\tests\unittests\CoreGraphics\CGColorTests.mm" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\CoreGraphics\CGImageTests.mm" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\CoreGraphics\TestUtils.mm" />
+  </ItemGroup>
+  <ItemGroup>
+    <SBResourceCopy Include="..\..\..\..\tests\unittests\CoreGraphics\images\**\*" />
```

Comment:
whats the idea here? SBResourceCopy is typically used to indicate what needs to be in the final SDK package which is different than what tests typically require.